File: 10 - 11. Introduction to Introductions/code/add_three_numbers.cpp
#include <iostream>
using namespace std;

int main() {
    int num1, num2, num3;

    int sum;

    cout << "Enter 1st number: ";
    cin >> num1;
    cout << " " << num1 << endl;

    cout << "Enter 2nd number: ";
    cin >> num2;
    cout << " " << num2 << endl;

    cout << "Enter 3rd number: ";
    cin >> num3;
    cout << " " << num3 << endl;

    sum = num1 + num2 + num3;

    cout << "Sum of the three numbers: " << sum << endl;

    return 0;
}

File: 10 - 11. Introduction to Algorithms/code/AddThreeNumbers.java
import java.util.Scanner;

public class AddThreeNumbers {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int num1, num2, num3, sum;

        System.out.println("Enter the first number: ");
        num1 = scanner.nextInt();

        System.out.println("Enter the second number: ");
        num2 = scanner.nextInt();

        System.out.println("Enter the third number: ");
        num3 = scanner.nextInt();

        sum = num1 + num2 + num3;

        System.out.println("Sum of the three numbers: " + sum);

        scanner.close();
    }
}